openlibrary/config.py

```diff
@@ -18,7 +18,7 @@ def load(config_file):
     """
     # for historic reasons
     global runtime_config
-    runtime_config = yaml.load(open(config_file))
+    runtime_config = yaml.safe_load(open(config_file))
```

Comment:
```suggestion
    with open(config_file) as in_file:
        runtime_config = yaml.safe_load(in_file)
```
Let's automate the file close() even if an Exception is raised by using the `with open() as` _context manager_.